### ISSUE TITLE
net_helpers.py calls deprecated Adam optimised

### ISSUE THREAD

**Issue by ariercole** [vanderschaarlab/clairvoyance#5]
In treatments/RMSN/libs/net_helpers.py the line

> def get_optimization_graph(loss, learning_rate, max_global_norm, global_step, optimisation_function=tf.train.AdamOptimizer):

tf.train.AdamOptimizer is deprecated: should now be tf.train.Adam()

i.e. 

> def get_optimization_graph(loss, learning_rate, max_global_norm, global_step, optimisation_function=tf.optimizers.Adam()):

**Comment by DrShushen** [comment]
Dependency-related issue - will be addressed in the overhaul of dependencies.
